index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
```

Comment:
**Setting The Viewport**
The viewport meta tag allows the browser to control pages dimension and scaling according to different device sizes.

### **Without Meta Tag**
![image](https://user-images.githubusercontent.com/68030297/185940442-ca8c4513-8332-4f4b-bec4-2766a2163298.png)

### **With meta Tag**
![image](https://user-images.githubusercontent.com/68030297/185940636-f132281a-c1cd-43a9-9a6b-6c066ed309c7.png)

- [ ] For that I highly recommend 👌👌👌 adding the viewport meta tag in between the `head` tag.